### PR TITLE
psp-1370 further updates as requested by Doug/Volker.

### PR DESCRIPTION
**Pull request by devinleighsmith**
Volker requested that the spatial data be split into two separate views. This also updates the FE to consume the new location view as Volker will be adding that to Geoserver today.

### CODE REVIEW

File: backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+     , PROP.ZONING,ZONING_POTENTIAL
+FROM   PIMS_PROPERTY       PROP                                                    INNER JOIN
+       PIMS_ADDRESS        ADDR ON ADDR.ADDRESS_ID        = PROP.ADDRESS_ID        INNER JOIN
+       PIMS_PROVINCE_STATE PROV ON PROV.PROVINCE_STATE_ID = ADDR.PROVINCE_STATE_ID INNER JOIN
+       PIMS_COUNTRY        CNTY ON CNTY.COUNTRY_ID        = ADDR.COUNTRY_ID
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+COMMIT TRANSACTION
```
Comment (Fosol) [inline backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql:62]: This is likely to cause failures when running the EF DB Migration.  Have you tested this with a fresh DB?

File: tools/keycloak/sync/appsettings.test.json [hunk 2]
```diff
@@ -24,6 +24,6 @@
   "RetryAttempts": 3,
   "AbortAfterFailure": 1,
   "Api": {
-    "Uri": "https://pims-test.apps.silver.devops.gov.bc.ca/api"
+    "Uri": "https://pims-app-test-3cd915-dev.apps.silver.devops.gov.bc.ca/api"
```
Comment (Fosol) [inline tools/keycloak/sync/appsettings.test.json:27]: Wish they would make friendly URLS.

File: backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql (hunk 1 above)
Comment (devinleighsmith) [reply backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql:62]: I've tested this with the previous version of this script.

Comment (devinleighsmith) [reply backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql:62]: trying now with a make db-clean.

Comment (devinleighsmith) [reply backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_BOUNDARY_VW.sql:62]: no issues.

File: tools/keycloak/sync/appsettings.test.json (hunk 2 above)
Comment (asanchezr) [reply tools/keycloak/sync/appsettings.test.json:27]: we do have friendly urls:
* https://dev-pims.th.gov.bc.ca/api
* https://tst-pims.th.gov.bc.ca/api
* https://uat-pims.th.gov.bc.ca/api

Comment (devinleighsmith) [reply tools/keycloak/sync/appsettings.test.json:27]: that's true, could update to use those instead.

Comment (asanchezr) [reply tools/keycloak/sync/appsettings.test.json:27]: yes please

**Review by asanchezr — APPROVED** [review]

**Pull request merged**